server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java

```diff
@@ -141,6 +141,7 @@ public void remove(String key) throws MessagingException {
         if (lock(key)) {
             try {
                 internalRemove(key);
+            } catch (MessagingException e) {
```

Comment:
I see your point. But this method seems to be more general than just for users. I'm not sure it is a good idea to ignore an exception when a mail is removed for example.